src/library/scala/jdk/OptionConverters.scala

```diff
@@ -120,33 +129,51 @@ object OptionConverters {
     case _ => Optional.empty[A]
   }
 
-  /** Convert a Scala `Option[Double]` to a Java `OptionalDouble` */
-  def toJavaOptionalDouble(o: Option[Double]): OptionalDouble = o match {
+  /** Convert a Scala `Option[Double]` to a Java `OptionalDouble`
+    *
+    * $primitiveNote
+    */
+  def toJavaOptionalDouble(o: Option[jl.Double]): OptionalDouble = o match {
     case Some(a) => OptionalDouble.of(a)
     case _ => OptionalDouble.empty
   }
 
-  /** Convert a Scala `Option[Int]` to a Java `OptionalInt` */
-  def toJavaOptionalInt(o: Option[Int]): OptionalInt = o match {
+  /** Convert a Scala `Option[Int]` to a Java `OptionalInt`
```

Comment:
```suggestion
  /** Convert a Scala `Option[Integer]` to a Java `OptionalInt`
```

or

```suggestion
  /** Convert a Scala `Option[java.lang.Interger]` to a Java `OptionalInt`
```

Maybe, for avoidance of doubt, the other ones should say also say `Option[java.lang.Long]`, etc?